### ISSUE TITLE
Option to disable horizontal line in "rainbow lines"

### ISSUE THREAD

**Issue by jibbers42** [tomasr/viasfora#324]
Hello, I would appreciate an option to disable the horizontal line in "rainbow lines", but keep the vertical one. I really like the feature, but unfortunately I find the horizontal line pretty distracting.

In VS Code this is possible and gives you something like:
![image](https://user-images.githubusercontent.com/797824/173818301-6b949dcc-2093-45d2-be9d-5bd5c6ef0ed0.png)

vs Viasfora's:
![image](https://user-images.githubusercontent.com/797824/173818475-f7659621-94ec-4d88-94cb-5df9fa5f60e3.png)

While example above isn't so bad, in busier code I don't like the horizontal line running through and creating a visual barrier between the lines of code.

Thanks!

**Comment by jibbers42** [comment]
Another example of the horizontal lines presence being a bit excessive:
![image](https://user-images.githubusercontent.com/797824/174011892-74fba507-b53b-4c82-82f9-783d98862528.png)

**Comment by realparadyne** [comment]
I really like the lines completely linking the brackets, so if it were changed, I would prefer it to become an option and not a change for everyone.

**Comment by tomasr** [comment]
@jibbers42 By "keeping the vertical line", do you mean something closer to Visual Studio's built-in scope guides?

**Comment by jibbers42** [comment]
Yes, I think if the built-in scope line were highlighted that would be fine.

Your plugin is a bit smarter though, for example I don't think the built-in guides understand multi-line parameter lists:
![image](https://user-images.githubusercontent.com/797824/188507614-b8a797ae-c2e5-4eac-92ec-4b1cba1dc112.png)

Unfortunately, your plugin does get confused with some indentation styles:
![image](https://user-images.githubusercontent.com/797824/188507640-4a79ad6b-8a41-4ee9-b4b6-0b4b07e7c1da.png)

So either choice of highlighting the built-in guides or removing the horizontals from your existing decoration would be great (and maybe fixing the indentation issue)!

**Comment by K-RutviK-K** [comment]
how to enable underline and vertical line visual studio 2019?
